tests/components/StandardFacets.stories.tsx

```diff
@@ -34,3 +34,23 @@ export const NoOptionCounts = (args: StandardFacetsProps) => {
     </SearchHeadlessContext.Provider>
   );
 };
+
+export const ShowMoreLimit = (args: StandardFacetsProps) => {
+  return (
+    <SearchHeadlessContext.Provider value={generateMockedHeadless(mockedHeadlessState)}>
+      <StandardFacets {...args} showMoreLimit={1} />
+    </SearchHeadlessContext.Provider>
+  );
+};
+
+export const ShowMoreLimitClicked = (args: StandardFacetsProps) => {
```

Comment:
since the returned JSX is the same as for `ShowMoreLimit`, I think you can just bind to that and add a `play` function. we do this in a couple other stories if you want to see an example